lib/diazo/rules.py

```diff
@@ -8,14 +8,22 @@
 
 import logging
 import re
+import sys
 
 from lxml import etree
+from lxml import html
 from six import string_types
 from future.moves.urllib.parse import urljoin
 
 from diazo.cssrules import convert_css_selectors
 from diazo.utils import namespaces, fullname, pkg_xsl, _createOptionParser
 
+# Conditional Imports
+if sys.version.startswith('2'):
+    from urllib2 import urlopen  # For Python 2
+else:
+    from urllib.requests import urlopen  # For Python 3
+
```

Comment:
I think this is what's causing the flake8 error. Diazo depends on the future module, so it should work to just import from the new location (may need to bump the minimum version for future in setup.py) http://python-future.org/standard_library_imports.html